manifests/0000_90_openshift-controller-manager-operator_04_operand-route-controller-servicemonitor.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
```

Comment:
This role is already defined in bindata/v3.11.0/openshift-controller-manager/route-controller-servicemonitor-role.yaml in https://github.com/openshift/cluster-openshift-controller-manager-operator/pull/255/commits/d505b981d68a79a02a0038dba60e40f6647b9a5e#diff-ddf2d7b29e1f246cc23d33358c5891d71becce69bd264c2495d5f8f607a2d3a6